README.md

```diff
@@ -310,14 +310,63 @@ $ ninja -C out/Release
 
 ### Running tests
 
-Run:
+#### Build
+
+Build (includes tests):
 
 ```bash
 $ ./gyp_uv.py -f make
 $ make -C out
+```
+
+#### Run all tests
+
+```bash
 $ ./out/Debug/run-tests
 ```
 
+This driver forks and executes tests one by one.
+
+#### Run one test
+
+```bash
+$ ./out/Debug/run-tests TEST_NAME
```

Comment:
I would add that running `$ ./out/Debug/run-tests TEST_NAME TEST_NAME` will run the test in the same process as the test runner, simplifying the use of debugging tools quite a lot. Also, point out that, tests that spawn processes won't work in this case.